### PR TITLE
Added the delete GUI

### PR DESCRIPTION
**Pull request by jblss888**

### CODE REVIEW

File: mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java [hunk 1]
```diff
@@ -119,7 +119,165 @@ else if(menuSelection.equals("Delete an existing record")){
...
+      panelThree.add(noButton);
+      
+     //This button is used to make a query to MySQL if user enters a valid product id
+      submitButton = new JButton("Submit");
+      submitButton.addActionListener(new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+            int fieldselection = 0;
+            //Wont go make a search until there is a valid product id
+            if(checkInteger(searchPid.getText())){
```
Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java:174]: I didn't see this method did you mean to add this one too?

File: mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java [hunk 2]
```diff
@@ -119,7 +119,165 @@ else if(menuSelection.equals("Delete an existing record")){
...
+      noButton.addActionListener(new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+           System.out.println("The record was NOT deleted");
+           cp.removeAll();
+           displayMenue();
+        }
+     });
+    
+      panelThree.add(noButton);
```
Comment (Naomimmm) [inline mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java:165]: Does the yes button also need to be added?

**Review by Naomimmm — APPROVED** [review]
Looks great

File: mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java (hunk 1 above)
Comment (jblss888) [reply mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java:174]: That's just for the button, its not not really a method by itself

File: mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java (hunk 2 above)
Comment (jblss888) [reply mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java:165]: The yes button is added later once the submit button is hit.

**Comment by jblss888** [comment]
There was some issues with one of my checks, its fixed now and you were right about adding the yes button. I though i had added it a bit later but I accidently deleted that add. I will make a new pull request with the working code

**Pull request closed without merging**